skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/serializer/AutomatesJSONSerializer.scala

```diff
@@ -173,25 +169,22 @@ object AutomatesJSONSerializer {
       case "MentionLocation" => new MentionLocationAttachment(json("filename").str, json("pageNum").arr.map(_.num.toInt), json("blockIdx").arr.map(_.num.toInt), attType)
       case "DiscontinuousCharOffset" => new DiscontinuousCharOffsetAttachment(json("charOffsets").arr.map(v => (v.arr.head.num.toInt, v.arr.last.num.toInt)), attType)
       case "ParamSetAtt" => new ParamSetAttachment(json("attachedTo").str, attType)
-      case "ParamSettingIntervalAtt" => {
+      case "ParamSettingIntervalAtt" =>
         var inclLower: Option[Boolean] = None
         var inclUpper: Option[Boolean] = None
         if (!json("inclusiveLower").isNull) inclLower = Some(json("inclusiveLower").bool)
```

Comment:
It looks like this can be `val inclusiveLowerOpt = json("inclusiveLower").boolOpt`.